scripts/stylelint/config/index.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+require('ts-node').register({
```

Comment:
This feels tremendously hacky.

I think it's be nicer to write the plugin in plain js, with [jsdoc comments](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) to provide typing. And then we'd be able to run this plugin as part of our regular stylelint run rather than needing a totally separate runtime

Reply:
Very hacky indeed! We introduced `ts-node` to enable access to the [Polaris tokens from within the stylelint config](https://github.com/Shopify/polaris-react/blob/79e358e5e9491bafe73a8f4a5d363453aacf2ebc/scripts/stylelint/config/config.ts#L1). Any suggestion on how we can import a `.ts` module in `.mjs`?